### ISSUE TITLE
[Feature Request] Hiding sticker on hover

### ISSUE THREAD

**Issue by Dragon-0609** [doki-theme/doki-theme-vscode#143]
**Is your feature request related to a problem? Please describe.**
It isn't such big problem, but it would be great if the problem will be solved. It is inconvenient when you can't see which terminal is active. I could increase height of the terminal, but that isn't point.

| Problem  |  How I solved.  |
|---|---|
|  ![vscode-terminal](https://user-images.githubusercontent.com/80271099/166148554-f02633a6-f8ab-4fb6-a196-398ea9a39f65.png) | ![vscode-terminal2](https://user-images.githubusercontent.com/80271099/166148643-525b9566-9750-4499-bd88-5dabdfbe61eb.png)  |

**Describe the solution you'd like**
When the user hovers the sticker, it'll be hidden till the user doesn't move the cursor away.

**Comment by Unthrottled** [comment]
Ah, yeah that's a good idea. I think I might add it as separate feature from the main sticker installation. You'll have to hover over the sticker for a short duration before it hides.

**Comment by Unthrottled** [comment]
After briefly looking into the issue, I forgot that I just put the stickers on a `css psuedo element`, because extension's cannot add things to the VSCode's DOM tree. 
https://github.com/doki-theme/doki-theme-vscode/blob/ce9522d4969030be73cc22745749b45dd3d51d3e/src/StickerService.ts#L138

So I'll probably need some more time to figure out how I want to get this to work. Adding a `:hover` effect to the parent doesn't work, because the parent is the entire VSCode window, so the sticker hides immediately.

**Comment by Dragon-0609** [comment]
> Adding a :hover effect to the parent doesn't work, because the parent is the entire VSCode window, so the sticker hides immediately.

What about `transition` and `opacity`?

```

.sticker {
      width: 186px;
      height: 218px;
      transition: 1.5s;
      background: url("asuna_dark.png") 0 0 no-repeat;
}
        
.sticker:hover {
      opacity: 0;
}

<div class="sticker"></div>

```

![sticker_hover](https://user-images.githubusercontent.com/80271099/166160845-b7c825f7-c694-4649-a267-6aa5ab833bc6.gif)

**Comment by Dragon-0609** [comment]
If you are able to get image properties such as height and width, it should work:

```

.sticker::after {
    position:absolute;z-index:9001;width:100%;height:100%;
}

.sticker > div::after {
    position: absolute;
    bottom: 0;
    right: 0;
    width: 186px;  /* WIDTH AND HEIGHT IS NECESSARY */
    height: 218px;
    transition: 1.5s;
    background-image: url("asuna_dark.png");
    content:'';
    
    background-repeat:no-repeat;opacity:1;
}

.sticker > div:hover::after {
    opacity: 0;
}

<div class="sticker">
    <div></div>
</div>

```

![sticker_hover](https://user-images.githubusercontent.com/80271099/166161517-a5d6267c-d721-42d8-b845-21ad80e9a317.gif)

**Comment by Unthrottled** [comment]
Yeah, that'd be nice if I could get the sticker put into a box the size of it, but right now the stickers are in this parent:

![image](https://user-images.githubusercontent.com/15972415/166162403-5c887947-1058-4ec1-9a9b-3d9894588547.png)

So I'll have to see if I can put them in a smaller parent, or some how inject it in.

**Comment by Apal4ah** [comment]
how to remove the sticker?
